src/runboat/settings.py

```diff
@@ -51,6 +64,8 @@ class Settings(BaseSettings):
     # A dictionary of variables to be set in the jinja rendering context for the
     # kubefiles.
     build_template_vars: dict[str, str] = {}
+    # The path of the default kubefiles to be used.
+    build_default_kubefiles_path: Path = Path(__file__).parent / "kubefiles"
```

Comment:
I generally try to prefer `importlib.resources` over `__file__` because it is often not defined if the application is packaged as a zipapp or similar.

```suggestion
    build_default_kubefiles_path: Path | None
```